### ISSUE TITLE
x/tools/internal/lsp/regtest: TestRegenerateCgo fails in some environments

### ISSUE THREAD

**Issue by findleyr** [golang/go#39135]
After syncing to head, TestRegenerateCgo is failing on @bcmills machine.  Relevant subsection of the logs (verbosity of test output was raised in #39130).

```
--- FAIL: TestRegenerateCgo (120.01s)
    --- FAIL: TestRegenerateCgo/singleton (60.00s)
        cgo_test.go:55: waiting on:
            	Unmet: empty diagnostics
            
            err:context deadline exceeded
            
            state:
            #### log messages (see RPC logs for full text):
            	Info: "2020/05/18 14:38:22 Build info\n----------\nversio...
            	Info: "2020/05/18 14:38:22 go env for /tmp/goplstest-sand...
            	Info: "2020/05/18 14:38:22 go/packages.Load\n\tsnapshot=0...
            	Info: "2020/05/18 14:38:23 go/packages.Load\n\tsnapshot=5...
            	Info: "2020/05/18 14:38:23 go/packages.Load\n\tsnapshot=5...
            	Info: "2020/05/18 14:38:52 background imports cache refre...
            	Info: "2020/05/18 14:38:52 background refresh finished af...
            
            #### diagnostics:
            	cgo.go (version 0):
            		(8, 9): fortytwo not declared by package C
            
            #### outstanding work:
        runner.go:202: Shutdown: Shutdown: context deadline exceeded
```

Something is racing: my guess is that this is a problem with detecting file changes.  I'll look into it, and in the meantime `t.Skip()` the test.

CC @heschik @stamblerre

**Comment by findleyr** [comment]
This turned out to be a problem with our internal GOPACKAGESDRIVER that affected both @bcmills and me. This also explains why it wasn't failing on the builders.

Closing in favor of our existing tracking bug.
